lua/nordic/groups/integrations.lua

```diff
@@ -348,17 +349,17 @@ function M.get_groups()
     G['@text.note'] = { link = 'Note' }
     --- Markup
     G['@markup'] = { link = '@none' }
-    G['@markup.emphasis'] = { italic = true }
+    G['@markup.emphasis'] = { fg = C.orange.base, italic = true }
     G['@markup.environment'] = { link = 'Macro' }
     G['@markup.environment.name'] = { link = 'Type' }
     G['@markup.heading'] = { link = 'Title' }
-    G['@markup.heading.1'] = { fg = C.yellow.base, bold = true }
-    G['@markup.heading.2'] = { fg = C.orange.base, bold = true }
-    G['@markup.heading.3'] = { fg = C.magenta.base, bold = true }
+    G['@markup.heading.1'] = { fg = C.yellow.base }
+    G['@markup.heading.2'] = { fg = C.orange.base }
+    G['@markup.heading.3'] = { fg = C.magenta.base }
     G['@markup.heading.4'] = { fg = C.green.base }
-    G['@markup.heading.5'] = { fg = C.blue2, italic = true }
-    G['@markup.heading.6'] = { fg = C.cyan.base, italic = true }
-    G['@markup.italic'] = { italic = true }
+    G['@markup.heading.5'] = { fg = C.blue2 }
+    G['@markup.heading.6'] = { fg = C.cyan.base }
```

Comment:
What about making all headings bold? I like that best, and that's what [tokyonight](https://github.com/folke/tokyonight.nvim) does.

I know the old bold and italic system is what GitHub uses, but when I make a heading, I am just trying to add another layer of structure, not make things italic.